docs/index.md

```diff
@@ -17,6 +17,7 @@
 ### Args
 
 - `paths`: A list of paths to files or directories to back up
+- `include_files`: A list of files containing include lists
```

Comment:
Can we name this `files _from` to match the restic flag name? I don't always keep it exactly matching, but I try to unless there's a good reason not to.